### PR TITLE
Views

### PR DESCRIPTION
**Pull request by jerryowusu**
## Description

In this milestone, you will be adding the Views associated with the Models and Controllers you've already created for your Blog App.

## General Requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

## Ruby requirements
- Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).

## Project requirements

- [x] Implement the design from the sneak peek wireframes.
- [x] Styling is nice to have but focus on making sure everything functions first.
- [x] Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- [x] Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

Have a nice time reviewing the code :+1:

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+      <p class="text-right"> Likes: <%= post.likes_counter ? post.likes_counter : 0  %>&nbsp</p>
+      <p class="text-right"> &nbsp Comments: <%= post.comments_counter ? post.comments_counter : 0 %></p>
+        </div>  
+      </div>
+
+    <%end%>
+  <div class="row">
+    <div class = "col-md-10 m-auto pb-5 d-flex justify-content-center" >
+      <div class= "mt-4">
+        <%= link_to "See all posts", user_posts_path(@user), class:'btn btn-xs btn-outline-primary' %>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb:48]: - [ ] I am not able to see all posts when I click on this link please address that

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+      <p class="text-right">Number of posts: 
+        <%= @user.posts_counter ?  @user.posts_counter : 0  %>
+      </p>
+    </div>  
+  </div>
+
+  <% @user.posts.each do |post| %>
+
+  <div class = "bg-light card col-md-9 m-1  p-3 d-flex" >
+    <h3>Post# <%=  post.id %> </h3>
```
Comment (DeliceLydia) [inline app/views/posts/index.html.erb:23]: - [ ] I am not able to see a post page to be able to see its comment see the error:
show

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @jerryowusu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for this project:
- Awesome README
- No linter errors


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (jerryowusu) [reply app/views/posts/index.html.erb:23]: Thanks @DeliceLydia, it has been resolved

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @jerryowusu, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] On your user show page the number of posts doesn't display. Please fix this
lili




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Comment by jerryowusu** [comment]
wow! I just missed an '=' to sign. Thanks a lot @keneogu

**Comment by jerryowusu** [comment]
@keneogu, would you please check and review again for me? Thanks

**Comment by jerryowusu** [comment]
@keneogu, I have been trying to reach you for further delibrations on the project

**Review by mikethreels — APPROVED** [review]
Hi Jerry, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jerryowusu** [comment]
Thank you  @mikethreels

**Pull request merged**